src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,95 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Person } from '../types/Person';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable/PeopleTable';
+import { PeopleFilters } from '../components/PeopleFilters/PeopleFilters';
+import { filteredPeople } from '../utils/filteredPeople';
+import { sortPeople } from '../utils/sortPeople';
+
+export const PeoplePage = () => {
+  const [searchParams] = useSearchParams();
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setLoading] = useState(false);
+  const sex = searchParams.get('sex');
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || null;
+  const sortType = searchParams.get('sort');
+  const order = searchParams.get('order') || null;
```

Comment:
Use an empty string instead of null for the default value of 'order' to avoid possible issues when using string methods or checking for truthy values.